Gruntfile.js

```diff
@@ -36,11 +36,18 @@ module.exports = function(grunt) {
 
         concat: {
             js: {
-                src: ['src/scripts/main.js', 'src/scripts/**/*.js'],
+                src: ['src/scripts/client/main.js', 'src/scripts/client/**/*.js'],
                 dest: 'build/tiyo-assistant.js',
                 options: {
                     sourceMap: true
                 }
+            },
+            worker: {
```

Comment:
Do we need this? Should we just use a `copy` task? I think you added it for a possible future worker... but is this premature?